SDWebImage/SDImageCache.h

```diff
@@ -132,6 +153,14 @@ typedef void(^SDWebImageCalculateSizeBlock)(NSUInteger fileCount, NSUInteger tot
 - (NSOperation *)queryDiskCacheForKey:(NSString *)key done:(SDWebImageQueryCompletedBlock)doneBlock;
 
 /**
+ * Query the disk cache asynchronously.
+ *
+ * @param key               The unique key used to store the wanted image
+ * @param returnDataForGIFs Return NSData for GIFs if YES
```

Comment:
mismatch param name, don't understand what to pass in here, a SDWebImageReturnConfig instance right>